### PR TITLE
Fix image urls

### PR DESCRIPTION
**Pull request by james77777778**
I've noticed that the image urls are broken.

Instead of:
`https://keras.io/home/hongyu/workspace/keras-io/guides/img/stable_diffusion_3_in_keras_hub/stable_diffusion_3_in_keras_hub_7_0.png`
The correct url should be:
`https://keras.io/img/guides/stable_diffusion_3_in_keras_hub/stable_diffusion_3_in_keras_hub_7_0.png`

The issue might be in `autogen.py` but I'm not sure what the root cause is.

CC @divyashreepathihalli 
I've fixed the image urls in SAM as well.

### CODE REVIEW

**Comment by divyashreepathihalli** [comment]
Thanks @james77777778 !!!

**Review by divyashreepathihalli — APPROVED** [review]

File: guides/ipynb/keras_hub/stable_diffusion_3_in_keras_hub.ipynb [hunk 1]
```diff
@@ -44,10 +44,10 @@
    },
    "outputs": [],
    "source": [
-    "!!pip install -Uq keras\n",
-    "!!pip install -Uq git+https://github.com/keras-team/keras-hub.git\n",
-    "!!wget --user-agent=\"User-Agent: Mozilla/5.0\" -O mountain_dog.png https://i.imgur.com/3AHYG9Z.png\n",
-    "!!wget --user-agent=\"User-Agent: Mozilla/5.0\" -O mountain_dog_mask.png https://i.imgur.com/n3Prpj6.png"
+    "!pip install -Uq keras\n",
+    "!pip install -Uq git+https://github.com/keras-team/keras-hub.git\n",
+    "!wget --user-agent=\"User-Agent: Mozilla/5.0\" -O mountain_dog.png https://i.imgur.com/3AHYG9Z.png\n",
```
Comment (fchollet) [inline guides/ipynb/keras_hub/stable_diffusion_3_in_keras_hub.ipynb:49]: This way of getting images from imgur isn't too great. How about... pushing the images to the keras.io repo (in an examples/assets/ folder or something like that) and linking them from github?

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

File: guides/ipynb/keras_hub/stable_diffusion_3_in_keras_hub.ipynb (hunk 1 above)
Comment (james77777778) [reply guides/ipynb/keras_hub/stable_diffusion_3_in_keras_hub.ipynb:49]: Got it. Will fix it in another PR!

**Pull request merged**